ArgsParsing/TypeParsers/UserParser.cs

```diff
@@ -26,9 +27,12 @@ public UserParser(IUserRepo userRepo)
         public override async Task<ArgsParseResult<User>> Parse(IImmutableList<string> args, Type[] genericTypes)
```

Comment:
this should also accept display names

Reply:
I know, I deliberately do not want to do that myself as that's a great first issue for someone else